miio/lib/devices/viomivacuum.js

```diff
@@ -159,7 +159,8 @@ module.exports = class extends Vacuum.with(
   }
 
   resumeCleanRooms(listOfRooms) {
-    return this.pause();
+    // TODO need to find out the miio command here.
```

Comment:
Not sure what's the miio command for viomivacuum. @afharo could you please help with this?

Reply:
I'm not sure either. Let's run the same command as the start room cleaning for now.